contract/bridge/contracts/OneBtc.sol

```diff
@@ -10,15 +10,19 @@ import {Redeem} from "./Redeem.sol";
 import {Replace} from "./Replace.sol";
 import {IRelay} from "./IRelay.sol";
 import "./IExchangeRateOracle.sol";
+import "./IVaultRegistry.sol";
 
 contract OneBtc is ERC20Upgradeable, Issue, Redeem, Replace {
     IRelay public relay;
+    IVaultRegistry public vaultRegistry;
+    IExchangeRateOracle oracle;
 
-    function initialize(IRelay _relay, IExchangeRateOracle _oracle) external initializer {
+    function initialize(IRelay _relay, IExchangeRateOracle _oracle, IVaultRegistry _vaultRegistry) external initializer {
```

Comment:
will this create an issue for contract upgrade? as upgrade don't work with new initializer? can you double check?

Reply:
Added functions to update the contract addresses in initialize() functions.